purchase_sale_inter_company/models/purchase_order_line.py

```diff
@@ -87,7 +87,10 @@ def write(self, vals):
         ).sudo()
         if not sale_lines:
             return res
-        closed_sale_lines = sale_lines.filtered(lambda x: x.state != "sale")
+        state = ["sale"]
+        if self.env.context.get("allow_update_locked_sales", False):
```

Comment:
Can you please add a test where you tests that you are able to write on po line with context key `allow_update_locked_sales` ?

Reply:
Hello @santostelmo , I updated